### ISSUE TITLE
abnormal localization of mutated protein

### ISSUE THREAD

**Issue by pfey03** [obophenotype/upheno#506]
We have a rather complex phenotype regime regarding 'localization' of cell parts or molecules. we have the GO conform abnormal localization of another protein to an organelle caused by the mutation, and also the direct abnormal localization of the mutated protein.  See the examples below/

Please read the **comments** of the terms

Parent of both branches:
DDPHENO:0001009 | aberrant localization

1. the GO conform branch: 

DDPHENO:0000471 | aberrant protein localization
def: "Deviation from the normal, usual, or expected localization of a protein." [DDB:pf, DDB:pg]
**comment**: Use for the altered localization of a protein that is affected by the mutation of another gene product.

typical children here are conform with GO, e.g.
GO:0072697 | protein localization to cell cortex
DDPHENO e.g.
DDPHENO:0001508 | abolished protein localization to cell cortex

2. The need for Dicty:

id: DDPHENO:0001217 | aberrant cellular localization
def: "Deviation from the normal, usual, or expected events during localization within the cell." [DDB:pf]
**comment**: Use for mutated gene products that do not localize to their typical cellular compartment compared to the wild type gene product.

So this is the mutated gene product’s localization itself. In GO the self cannot be annotated but as a phenotype it makes sense.

Example
DDPHENO:0000062 | aberrant localization to coated pits. 

The terms could be renamed if needed. 

Do any others annotate phenotypes where the mutated protein mislocalizes to a cell part?

Thanks!

**Comment by mah11** [comment]
Yes, we have lots of terms in FYPO for abnormal protein localization, and _lots_ of annotations using them.

The subtypes are differentiated by increased/decreased/abolished, and by where the protein would normally go (e.g. decreased protein localization to nucleus). We don't distinguish localization of a mutant protein from localization of some other protein using separate terms. Instead, we use the same localization phenotype term in either case, and put the mislocalized protein in an annotation extension, which can be the "self" or a different protein (and for now we're using gene IDs).

We also have a set of "normal protein localization to x" terms, and similarly use extensions to identify the normally localized proteins in annotations.

**Comment by mah11** [comment]
You can see some examples of FYPO terms, and annotations with "self" and "non-self" extensions on PomBase ontology term pages, e.g. for decreased protein localization to nucleus:

https://www.pombase.org/term/FYPO:0004455

**Comment by matentzn** [comment]
@mah11 Can you suggest a logical definition to capture this? I can then move on to create the pattern.

**Comment by mah11** [comment]
Well, I'll try, but this is an area that I don't think is completely sorted yet anyway ...

The logical definitions currently in FYPO mostly use GO biological processes from the "protein localization" branch. I'm not at all convinced that we shouldn't change this approach, though, because there are lots of locations where proteins can normally (or abnormally) be found, but for which there probably isn't a localization process that's genuinely distinct from the general protein localization case. There are indications that GO editors would eventually like to review the protein localization subclasses and purge a lot of them; for example, see the comment thread in https://github.com/geneontology/go-ontology/issues/16695

In cases where there's a bona fide "protein localization to X" process, the abnormalBiologicalProcess pattern should still work, e.g.:

example term: abnormal protein localization to nucleus

"'has_part' some ('process quality' and ('inheres_in' some 'protein localization to nucleus') and ('has_modifier' some 'abnormal'))"

I'm less sure what to do for the cases where a given location X doesn't have its own "localization to" process - maybe something using abnormalLevel, e.g.:

example term: decreased protein localization to polysome

"'has_part' some ('decreased amount' and ('inheres_in' some ('protein' and ('part_of' some 'polysome'))) and ('has_modifier' some 'abnormal'))"

... but I am absolutely guessing, and so welcome improvements! (note: the example is based on a FYPO term that doesn't have a logical def at preseny.)

In each case, whatever edits ensue, I also envision parallel "normal", "abnormally increased", and "abnormally decreased" patterns for the "abnormal" versions shown.

I also don't know how to indicate the annotation extensions that we use to identify the protein that gets normally/abnormally localized. What we're using now is a home-brewed "assayed_using" relation and PomBase gene IDs; we're aware that the latter is at best an oversimplification. In particular, we don't have a good way to capture the (mis)localization of a mutant protein - we use the same gene ID whether the protein is wild type or mutant, and if it's mutant we stick a comment to that effect in a free-text field that's only visible in Canto at present. Not great!

assayed_using(PomBase:id)

**Comment by pfey03** [comment]
At today's call, we decided to follow pombase's way to post-compose terms with the affecting protein in the extension. See link above from pombase. Thanks Midori for sharing the link.

For new pattern, the PATO term mislocalized PATO:0000628 will be used. At dictyBase we will unify our two branches of misloalizations and in our new database also post-compose with specific affecting gene/protein.
